### ISSUE TITLE
Up/Down arrow keys not working when JAWS-17 is open

### ISSUE THREAD

**Issue by ShawnXu** [select2/select2#4068]
Here is a screencast http://screencast.com/t/ROFpa3ZhGyd recorded in Chrome with JAWS 17 open on https://select2.github.io/examples.html

Our QA reported that when JAWS is on, the Up/Down arrow keys can fail to navigate.  Pressing the Down arrow key or TAB key sometimes results in "document end" according to JAWS read, which may have something to do with the fact that select2's option list is appended to the bottom of the page. Not sure if there could a a fix on select2's side. 

I'm also bring this issue up to JAWS support since it's possible that it's their software intercepting and interrupting those key strokes.

Thanks,
Shawn

**Comment by ShawnXu** [comment]
Hi, I wonder if select2 dev had the bandwidth to look at this issue yet?

**Comment by kevin-brown** [comment]
As with most accessibility issues, we don't have the tools necessary to be able to quickly debug them. So any debugging comes during the short periods of time where we have access to JAWS (or other screen readers, when they are properly configured), assuming we don't have other things that take higher priority.

My _guess_ is that JAWS is intercepting the keys before they reach Select2, but I have no idea what can be done to prevent that (on Select2's side, at least).

**Comment by krtek4** [comment]
Hi There,

Currently having the same issue with Jaws 17 on Windows 10 with IE 11.

What I remarked is that even with Jaws 17, the first example on the example page https://select2.github.io/examples.html#single does not work at all with the keyboard if it was selected using "tab".

Steps to reproduce :

1. Put the focus on the "real select"
2. Press Tab so the focus goes to the select2 version
3. Press Up / Down

Nothing is selected and the page just scroll around. Arrow key selection seems to work fine withtout Jaws when the select2 is opened using the mouse, but fails with Jaws, so this seems to be a "compound" issue.

**Comment by vlad-saling** [comment]
Hey, we are also experiencing this issue. Symptom and steps to reproduce match. I'll talk to our devs if we can give Select2 team a hand with this. Although I am not exactly sure where to begin.

**Comment by alexweissman** [comment]
Has anyone been able to confirm if v4.0.4 resolves this issue?  #4427 fixed a major focus issue.
